### DISCUSSION TITLE
Grid/Row gap and how to add vertical spacing

### DISCUSSION THREAD

**Discussion by rolznz** [heroui-inc/heroui#832] [Feedback]
Why not just use flex-gap? the gap on the grid seems useless because it applies to the sides of the grid, rather than, what if you want gaps between the rows, which seems to be the most common usage? do you expect a user to add a spacer element manually on each row?

If I manually add flex-gap, how can I ensure I use the same spacing units as the application default styles?

Sandbox:
https://codesandbox.io/s/sandpack-project-forked-jekpd8

What I would expect to happen would be a vertical gap of 1 unit between the 2 rows.

**Comment by Xariwey** [comment]
this is what is used to calc the gap
```js
`calc(${gap} * $space$sm)`
```

just to give u more info 
```js
space: {
    sm: "0.75rem",
```
